src/WingetCreateCore/Models/InstallerUpdateHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.WingetCreateCore.Models
+{
+    using System.Collections.Generic;
+    using Microsoft.WingetCreateCore.Models.Installer;
+
+    /// <summary>
+    /// Helper class for storing information relating to updating an installer.
+    /// </summary>
+    public class InstallerUpdateHelper
```

Comment:
>InstallerUpdateHelper [](http://example.com/codeflow?start=17&length=21)

nit: I don't love this name. Helper to me implies something that exposes functionality to do things, this is really just a POCO class describing metadata about a url/installer. I think something like InstallerMetadata or InstallerUrlMetadata would be more apt?

Reply:
Thanks for the suggestion, I was struggling to find a good name for this class. I changed the naming to "InstallerMetadata"